src/opm/input/eclipse/share/keywords/000_Eclipse100/T/TABDIMS

```diff
@@ -56,7 +56,7 @@
     },
     {
       "item": 9,
-      "name": "NUM_EOS_RES",
+      "name": "NUM_STATE_EQ",
```

Comment:
Changed to this name is because there are many usage of the name `NUM_STATE_EQ`, while no usage of the name `NUM_EOS_RES`.

Reply:
I do not have strong feeling to either.  `RES` is difficult to be related to the number of regions to me, not sure about  what others think. 

Basically, this item means the number of EOS regions, or maybe we can also interpret as number of equation of states.  

Maybe follow other items (https://github.com/OPM/opm-common/blob/94419eecf76d29f19f7f697dc902118ec50401e2/src/opm/input/eclipse/share/keywords/000_Eclipse100/T/TABDIMS#L79C18-L79C18), we can use `NUM_EOS_REGIONS` ?

Reply:
> `RES` is difficult to be related to the number of regions to me

Just to be clear, `RES` does not mean "regions".  It means "reservoir conditions".  That's also why we have `NUM_EOS_SURFACE` for the surface conditions.  I suppose a compromise might be

* NUM_EOS_REG_RESERVOIR
* NUM_EOS_REG_SURFACE

instead.

Reply:
Thanks for the clarification. Then I think we should go back to use `NUM_EOS_RES`. Will update the PR.